initfiles/componentfiles/configxml/mysqlserver.xsd

```diff
@@ -116,19 +116,25 @@
                     </xs:appinfo>
                 </xs:annotation>
             </xs:attribute>
-            <xs:attribute name="computer" type="computerType" use="required">
+            <xs:attribute name="computer" type="computerType" use="optional">
                 <xs:annotation>
                     <xs:appinfo>
                         <tooltip>The computer on which the MySQL database is deployed.</tooltip>
                         <autogenforwizard>1</autogenforwizard>
                     </xs:appinfo>
                 </xs:annotation>
             </xs:attribute>
+            <xs:attribute name="host" type="xs:string" use="optional">
+                <xs:annotation>
+                    <xs:appinfo>
+                        <tooltip>The host name on which the MySQL database is deployed. This is used before computer option.</tooltip>
```

Comment:
Your comment says "This is used before computer option", but if I'm reading the code above correctly it uses host if computer=''?  Which would be "This is used if computer is not set".